pom.xml

```diff
@@ -1624,6 +1624,8 @@
                   <exclude>activemq-artemis-native/src/main/c/org_apache_activemq_artemis_jlibaio_LibaioContext.h</exclude>
                   <exclude>**/dependency-reduced-pom.xml</exclude>
 
+                  <!-- Mockito -->
+                  <exclude>**/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker</exclude>
```

Comment:
what is this? (simple question. .no review intended here)

Reply:
This file is being used to allow Mockito to use a special new engine that make possible to override method calls of final classes